homeassistant/components/matter/climate.py

```diff
@@ -188,6 +188,7 @@ class MatterClimate(MatterEntity, ClimateEntity):
     _attr_hvac_mode: HVACMode = HVACMode.OFF
     _feature_map: int | None = None
     _enable_turn_on_off_backwards_compatibility = False
+    _postfix_translation_key = "thermostat"
```

Comment:
Can you tell me a bit what this does ? Is this a new feature ?

Reply:
It's a fallback translation key. It will be only used when we have primary attribute on multiple endpoints (when we suffix the name by the endpoint id).
It's used here https://github.com/home-assistant/core/pull/127798/files#diff-b6c426ab993a9279a458078576e5fa2a2a8746b1f97ee3ae583877b2fae9cb57R87-R88